saleor/graphql/webhook/enums.py

```diff
@@ -41,6 +41,9 @@
     WebhookEventAsyncType.ACCOUNT_CHANGE_EMAIL_REQUESTED: (
         "Account email change is requested."
     ),
+    WebhookEventAsyncType.ACCOUNT_SET_PASSWORD_REQUESTED: (
+        "Account set password is requested."
```

Comment:
It sounds unnatural, I would rephrase it to something like "Setting a new password for the account is requested."